Tests/WorkspaceTests/InitTests.swift

```diff
@@ -14,458 +14,422 @@ import Basics
+                try verifyTestFileContents(at: path, name: name, hasSwiftTesting: true, hasXCTest: true)
+
+                #if canImport(TestingDisabled)
+                let expectedPath = path.appending(components: ".build", Self.targetTriple.platformBuildPathComponent, "debug", "Modules", "\(name).swiftmodule")
+                expectFileExists(at: expectedPath)
+                #endif
+            }
+        )
+    }
 
-            // Verify basic file system content that we expect in the package
-            let manifest = path.appending("Package.swift")
-            XCTAssertFileExists(manifest)
-            let manifestContents: String = try localFileSystem.readFileContents(manifest)
-            XCTAssertMatch(manifestContents, .and(.contains(".plugin("), .contains("targets: [\"MyCommandPlugin\"]")))
-            XCTAssertMatch(manifestContents, .and(.contains(".plugin("),
-                .and(.contains("capability: .command(intent: .custom("), .contains("verb: \"MyCommandPlugin\""))))
-
-            // Check basic content that we expect in the plugin source file
-            let source = path.appending("Plugins", "MyCommandPlugin.swift")
-            XCTAssertFileExists(source)
-            let sourceContents: String = try localFileSystem.readFileContents(source)
-            XCTAssertMatch(sourceContents, .contains("struct MyCommandPlugin: CommandPlugin"))
-            XCTAssertMatch(sourceContents, .contains("performCommand(context: PluginContext"))
-            XCTAssertMatch(sourceContents, .contains("import XcodeProjectPlugin"))
-            XCTAssertMatch(sourceContents, .contains("extension MyCommandPlugin: XcodeCommandPlugin"))
-            XCTAssertMatch(sourceContents, .contains("performCommand(context: XcodePluginContext"))
-        }
+    /// Tests creating packages with no testing libraries.
+    @Test(arguments: [InitPackage.PackageType.library, .executable, .tool], [BuildSystemProvider.Kind.native, .swiftbuild])
+    func initPackageWithNoTests(packageType: InitPackage.PackageType, buildSystem: BuildSystemProvider.Kind) async throws {
+        #if canImport(TestingDisabled)
+            let buildSys = buildSystem
+        #else
+            let buildSys: BuildSystemProvider.Kind? = nil
+        #endif
+
+        try await createAndVerifyPackage(
+            packageType: packageType,
+            supportedTestingLibraries: [],
+            buildSystem: buildSys,
+            customVerification: { path, name in
+                let manifestContents: String = try localFileSystem.readFileContents(path.appending("Package.swift"))
+                #expect(!manifestContents.contains(#".testTarget"#))
+
+                expectDirectoryDoesNotExist(at: path.appending("Tests"))
+
+                #if canImport(TestingDisabled)
+                let expectedPath = path.appending(components: ".build", Self.targetTriple.platformBuildPathComponent, "debug", "Modules", "\(name).swiftmodule")
```

Comment:
issue (blocking):  the `expectedPath` assume the native build system, which is not the same as the SwiftBuild build system.  it's also only looking at `debug` build configuration.  But doesn't this test also build `release` build configuration?   So why are we only looking for the `debug` swift module and there is no expectation on the `release` build configuration?